docs/content/concepts/assets/asset-auto-execution.mdx

```diff
@@ -71,62 +79,99 @@ defs = Definitions(
+
+```python file=/concepts/assets/auto_materialize_after_all_parents.py
+from dagster import AutoMaterializePolicy, AutoMaterializeRule, asset
+
+wait_for_all_parents_policy = AutoMaterializePolicy.eager().with_rules(
+    AutoMaterializeRule.skip_on_not_all_parents_updated()
+)
+
+
+@asset(auto_materialize_policy=wait_for_all_parents_policy)
+def asset1(upstream1, upstream2):
     ...
+```
 
+#### Auto-materialize even if some parents are missing
 
-@asset(
-    auto_materialize_policy=AutoMaterializePolicy.lazy(),
-    freshness_policy=FreshnessPolicy(maximum_lag_minutes=24 * 60),
-    deps=[asset1],
+By default, the eager policy will refuse to materialize an asset if any of its parents are missing. In some cases, it's desirable to allow the downstream asset to be materialized, even if some of its parent assets / partitions are missing. To enable this, the `skip_on_parent_missing` rule can be removed from a given policy to prevent this from blocking materialization of an asset.
+
+```python file=/concepts/assets/auto_materialize_with_missing_parents.py
+from dagster import asset, AutoMaterializePolicy, AutoMaterializeRule
+
+allow_missing_parents_policy = AutoMaterializePolicy.eager().without_rules(
+    AutoMaterializeRule.skip_on_parent_missing(),
 )
-def asset2():
+
+
+@asset(auto_materialize_policy=allow_missing_parents_policy)
+def asset1(upstream1, upstream2):
     ...
 ```
 
-Freshness policies express how fresh an asset needs to be relative to data at the root of the graph. This means that, for an asset to meet it's freshness policy, both it and upstream assets need to be materialized in time.
+### Auto-materialization and partitions
 
-Setting a lazy auto-materialize policy on an asset allows it to be auto-materialized to help downstream assets meet their freshness policies. In this example, both `asset2` and `asset3` will be auto-materialized up to once per day, to help `asset3` meet its freshness policy. Conversely, if `asset2`, did not have an auto-materialize policy, then `asset3` would never become fresh, unless `asset2` were materialized in some other way.
+<PyObject object="AutoMaterializeRule" pluralize /> generally apply individually
+to each partition of a partitioned asset. Here's a pipeline with two daily-partitioned
+assets that have eager auto-materialize policies. At the end of each day, a partition
+for that day will be added to the set of partitions for each of the assets. Dagster
+will notice that the new partitions exist, but have no materializations, and then
+auto-materialize them.
```

Comment:
haha yeah no clue what happened there either

Reply:
@OwenKephart I just realized that I think I know what happened. When you start a paragraph with a `PyObject`, the `make mdx-format` script will throw a fit because... reasons. If you change this to start with a regular ol' character, remove the newlines, and then run the script again, it *should* behave.